Java/validation/src/main/java/com/lantanagroup/link/validation/configs/BlobStorageConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.lantanagroup.link.validation.configs;
+
+import com.lantanagroup.link.validation.services.BlobStorageService;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Getter
+@Setter
+@Configuration
+@ConfigurationProperties("blob-storage")
```

Comment:
Where is this configuration stored? In Azure I suppose, do we want to keep it in application files too